modules/terraform-aws-alternat/variables.tf

```diff
@@ -162,3 +162,45 @@ variable "vpc_id" {
   description = "The ID of the VPC."
   type        = string
 }
+
+variable "lambda_package_type" {
+  description = "The lambda deployment package type. Valid values are \"Zip\" and \"Image\". Defaults to \"Image\"."
+  type        = string
+  default     = "Image"
+  nullable    = false
+}
+
+variable "lambda_memory_size" {
+  description = "Amount of memory in MB your Lambda Function can use at runtime. Defaults to 256."
+  type        = number
+  default     = 256
+}
+
+variable "lambda_timeout" {
+  description = "Amount of time your Lambda Function has to run in seconds. Defaults to 300."
+  type        = number
+  default     = 300
+}
+
+variable "lambda_handlers" {
+  description = "Lambda handlers."
+  type = object({
+    connectivity_tester       = string,
+    alternat_autoscaling_hook = string,
+  })
+  default = {
+    connectivity_tester       = "app.connectivity_test_handler",
+    alternat_autoscaling_hook = "app.handler"
+  }
+}
+
+variable "lambda_environment_variables" {
```

Comment:
Out of curiosity, what is the intended use for this?

Reply:
I thought it would be nice to future-proof ourselves with an option to add custom environmental variables, for example to overwrite defaults (if more than one given map or object defines the same key or attribute, then the one that is later in the argument sequence takes precedence by `merge`) or supply some Python-specific configuration, like logging if there's going to be one, etc.